Dockerfile

```diff
@@ -20,8 +24,8 @@ ENV RAILS_ENV=$RAILS_ENV
 
 RUN bundle config --global jobs `cat /proc/cpuinfo | grep processor | wc -l | xargs -I % expr % - 1` && \
     if echo "development test" | grep -w "$RAILS_ENV"; then \
-    bundle install; \
-    else bundle install --without development test; fi
+    gem update --system && bundle install; \
```

Comment:
Just curious: what did you run into that made this necessary? I would have guessed that rubygems would have been up to date on the 2.7 image. Is it because the bundler version used was ahead?

Reply:
> . Is it because the bundler version used was ahead?

Right. Yeah I ran `docker-compose run --rm app bin/rails db:setup` and ran into a warning that bundler version was still 2.1.4.